ddtrace/llmobs/_llmobs.py

```diff
@@ -75,30 +95,103 @@ def _stop_service(self) -> None:
             log.warning("Failed to shutdown tracer", exc_info=True)
 
     @classmethod
-    def enable(cls, tracer=None):
+    def enable(
+        cls,
+        ml_app: Optional[str] = None,
+        integrations: Optional[list[str]] = None,
+        apm_enabled=False,
+        dd_env: Optional[str] = None,
+        dd_service: Optional[str] = None,
+        dd_site: Optional[str] = None,
+        dd_api_key: Optional[str] = None,
+        tracer=None,
+    ):
+        """
+        Enable LLM Observability tracing.
+
+        :param str ml_app: The name of your ml application.
+        :param list[str] integrations: A list of integrations to enable auto-tracing for.
+        :param str dd_site Your datadog site.
+        :param str dd_api_key: Your datadog api key.
+        :param str dd_apm_enabled: Whether Datadog Application Performance Monitoring is enabled.
+        """
         if cls._instance is not None:
             log.debug("%s already enabled", cls.__name__)
             return
 
+        if dd_api_key:
+            config._dd_api_key = dd_api_key
+
         if not config._dd_api_key:
             cls.enabled = False
             raise ValueError(
                 "DD_API_KEY is required for sending LLMObs data. "
                 "Ensure this configuration is set before running your application."
             )
+
+        # update environment config based on APM enabled/disabled
+        os.environ.update(cls._apm_env_config if apm_enabled else cls._no_apm_env_config)
+
+        if ml_app:
+            config._llmobs_ml_app = ml_app
+        if dd_site:
+            config._dd_site = dd_site
+        if dd_api_key:
+            config._dd_api_key = dd_api_key
+        if dd_env:
+            config.env = dd_env
+        if dd_service:
+            config.service = dd_service
+
         if not config._llmobs_ml_app:
             cls.enabled = False
             raise ValueError(
                 "DD_LLMOBS_APP_NAME is required for sending LLMObs data. "
                 "Ensure this configuration is set before running your application."
             )
 
+        # enable LLMObs integations
+        llmobs_integrations = {
+            LLMObs.langchain: lambda: LLMObs._patch_langchain(),
+            LLMObs.openai: lambda: LLMObs._patch_openai(),
+            LLMObs.botocore: lambda: LLMObs._patch_bedrock(),
+        }
+        if integrations:
+            for integration in integrations:
+                if integration in llmobs_integrations:
+                    llmobs_integrations[integration]()
+                else:
+                    log.warning(
+                        "%s is unsupported - LLMObs currently supports %s", integration, str(llmobs_integrations)
+                    )
+
         cls._instance = cls(tracer=tracer)
         cls.enabled = True
         cls._instance.start()
         atexit.register(cls.disable)
         log.debug("%s enabled", cls.__name__)
 
+    @classmethod
```

Comment:
introduce these functions so that customers can manually turn on llmobs integrations with the LLMObs service without importing any other ddtrace related code